packages/core/src/browser/window/window-service.ts

```diff
@@ -35,7 +35,7 @@ export interface WindowService {
      * Opens a new default window.
      * - In electron and in the browser it will open the default window without a pre-defined content.
      */
-    openNewDefaultWindow(params?: WindowSearchParams): void;
+    openNewDefaultWindow(params?: { search?: WindowSearchParams, hash?: string }): void;
```

Comment:
Suggestion: We should have a proper type for this. Something like:

```ts
export interface WindowOptions {
  search?: WindowSearchParams
  hash?: string
}
```

```suggestion
    openNewDefaultWindow(options?: { search?: WindowSearchParams, hash?: string }): void;
```